AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift

```diff
@@ -179,33 +281,25 @@
                 return didFail(with: error, completionHandler: completionHandler)
             }
             
-            if delegatedAuthenticationState.isDeviceRegistrationFlow {
-                performDelegatedRegistration(token.delegatedAuthenticationSDKInput) { [weak self] result in
-                    self?.deliver(challengeResult: challengeResult,
-                                  delegatedAuthenticationSDKOutput: result.successResult,
-                                  completionHandler: completionHandler)
-                }
+            if shouldShowRegistrationScreen {
+                presenter.showRegistrationScreen(
+                    component: self,
+                    registerDelegatedAuthenticationHandler: { [weak self] in guard let self = self else { return }
```

Comment:
maybe a new line here after the `in` :-)

Reply:
Done. Thanks ✅